manifests/i/IrfanSkiljan/IrfanView-plugins/4.58/IrfanSkiljan.IrfanView-plugins.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: IrfanSkiljan.IrfanView-plugins
+PackageVersion: 4.58
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+Installers:
+  - Architecture: x64
+    InstallerType: exe
+    InstallerUrl: https://download.betanews.com/download/1099412658-1/iview458_plugins_x64_setup.exe
```

Comment:
```
    InstallerUrl: https://download.betanews.com/download/1099412658-1/iview458_plugins_x64_setup.exe
```
cc @OfficialEsco: do we want to be using the betanews mirror here? I think I might have saw someone removing manifests with betanews URLs because they claimed that it was shipping "spyware" or "bloatware" or something.

Reply:
Unfortunately there is no canonical file download location listed, and though I haven't checked them all, other download locations seem to have temporary download links. The reason for using betanews is simple: I copied it from IrfanSkiljan.IrfanView.

We could try reaching out to Irfan (the original software author) for a better download location?

Reply:
Actually never mind; the removed manifests were about TechSpot URLs. BetaNews seems fine to me.